tests/src/unit/tests/test_connection.cpp

```diff
@@ -190,6 +216,60 @@ TEST_F(ConnectionUnitTest, Ssl) {
   EXPECT_EQ(state.status, STATUS_SUCCESS);
 }
 
+TEST_F(ConnectionUnitTest, SslDefaultVerifyPaths) {
+  const String host = "127.0.0.1";
+  const int verification_flags = CASS_SSL_VERIFY_PEER_CERT | CASS_SSL_VERIFY_PEER_IDENTITY;
+  const String cert_path = "tmp.cassandra.unit-test.cert";
+
+  mockssandra::SimpleCluster cluster(simple());
+  const String cert = cluster.use_ssl(host);
+  EXPECT_FALSE(cert.empty()) << "Unable to enable SSL";
+  ConnectionSettings settings;
+  settings.socket_settings.ssl_context = SslContextFactory::create();
+  settings.socket_settings.ssl_context->set_verify_flags(verification_flags);
+  ASSERT_EQ(cluster.start_all(), 0);
+
+  // Test that cert verification fails prior to calling set_default_verify_paths
+  Connector::ConnectionError connect_rc = Connector::CONNECTION_OK;
+  Connector::Ptr connector0(new Connector(Host::Ptr(new Host(Address(host, PORT))),
+                                          PROTOCOL_VERSION,
+                                          bind_callback(on_connection_error_code, &connect_rc)));
+  connector0->with_settings(settings)->connect(loop());
+  uv_run(loop(), UV_RUN_DEFAULT);
+  EXPECT_EQ(connect_rc, Connector::CONNECTION_ERROR_SSL_VERIFY)
+      << "Verification succeeded without certificate.";
+
+  const String cwd = current_dir();
+
+  // Generate certificate as file (which is used by our mock cluster) and import it
+  std::ofstream cert_buffer(cert_path.c_str());
+  cert_buffer << cert;
+  cert_buffer.close();
+  setenv("SSL_CERT_FILE", cert_path.c_str());
+  setenv("SSL_CERT_DIR", cwd.c_str());
+  std::cout << "Debug SslDefaultVerifyPaths: SSL_CERT_FILE " << cert_path << " " << cert << std::endl;
+  for (const auto var: {"SSL_CERT_FILE", "SSL_CERT_DIR"}) {
```

Comment:
This newer style of "range-based for" doesn't seem to be supported on very old versions like MSVC10. On VC11 it seems to be failing on the inline definition of the array. Maybe try something like this?
```suggestion
  const char *sslvarnames[] = {"SSL_CERT_FILE", "SSL_CERT_DIR", nullptr};
  for (int i = 0; sslvarnames[i]; i++) {
    const char *var = sslvarnames[i];
```